backend/app/api/routes/chat.py

```diff
@@ -229,3 +229,14 @@ def get_recommend_questions(session: SessionDep, chat_message_id: int):
 
     return get_chat_message_recommend_questions(session, chat_message)
 
+
+@router.post("/chat-messages/{chat_message_id}/recommend-questions", response_model=List[str])
+def refresh_recommend_questions(session: SessionDep, chat_message_id: int):
```

Comment:
How about using an adj "recommended" ?

Reply:
@634750802 The front-end needs to change it as well.